app/services/subscriptions/create_service.rb

```diff
@@ -32,6 +34,8 @@ def create(**args)
         organization_id: args[:organization_id],
         id: args[:plan_id]&.strip,
       )
+      @name = args[:name]&.strip
```

Comment:
Just a suggestion : what about passing the `args` to  the `process_create`?
It would allow you to add the striping logic in only one place, could make it easier to add new arguments in the future

Reply:
Can you please explain this one a bit more? :)

Reply:
Sorry, I wasn't clear :)

My suggestion is to change the signature of the `process_create` method to pass all updatable attributes:

```ruby
def process_create(args)
  #  ...
end
```